src/ussdapp.js

```diff
@@ -174,23 +179,30 @@ go.app = function() {
                 .get_quiz_question(self.im, self.im.user.answers.quiz_status.questions_remaining[0])
                 .then(function(quiz_question) {
                     var correct_answer = go.utils_project.get_correct_answer(quiz_question.answers);
+
                     return new ChoiceState(name, {
                         question: quiz_question.question,
                         choices: go.utils_project.construct_choices(quiz_question.answers),
                         next: function(choice) {
                                 var response_text = "";
-                                if (choice.value === correct_answer) {
+                                var correct = choice.value === correct_answer;
+                                if (correct) {
                                     response_text = quiz_question.response_correct;
                                     self.im.user.answers.quiz_status.questions_answered.push({"question": quiz_question.question, "correct": true});
                                 } else {
                                     response_text = quiz_question.response_incorrect;
                                     self.im.user.answers.quiz_status.questions_answered.push({"question": quiz_question.question, "correct": false});
                                 }
 
-                                return  {
-                                    name: "state_response",
-                                    creator_opts: response_text
-                                };
+                                return go.utils_project
+                                    .log_quiz_answer(self.im, quiz_question, choice.value, choice.label,
+                                        correct ? "True" : "False", response_text, self.im.user.answers.tracker)
```

Comment:
this should just pass `correct` in directly. The JSON parser should handle it ok.